Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -91,20 +91,29 @@ class DisplayHymnViewModel: ObservableObject {
                     self.chordsUrl = chordsPath.flatMap({ path -> URL? in
                         HymnalNet.url(path: path)
                     })
+                    if self.chordsUrl != nil {
```

Comment:
prefer optional unwrapping vs checking for nil: https://www.hackingwithswift.com/sixty/10/2/unwrapping-optionals

Reply:
Resolved in recent commit